agi_avdpool_sogis/agi_dm01avso24_bdbed.sql

```diff
@@ -0,0 +1,107 @@
+SELECT
+    boflaeche.geometrie AS wkb_geometry,
+    CASE
```

Comment:
Für diesen einfachen Fall gäbe es übrigens eine etwas einfachere Variante von CASE:

```
CASE boflaeche.art
  WHEN 'Gebaeude' THEN 0
  WHEN 'befestigt.Strasse_Weg' THEN 1
  WHEN 'befestigt. Trottoir' THEN 3
...

```
Siehe Abschnitt "Simple CASE" unter https://www.postgresql.org/docs/11/plpgsql-control-structures.html#PLPGSQL-CONDITIONALS